### ISSUE TITLE
Need nidigital system tests

### ISSUE THREAD

**Issue by texasaggie97** [ni/nimi-python#986]
### Description of issue

Once nidigital is merged into master and the API is better defined, we need to add system tests to ensure coverage.

Some API areas that need to be tested (List is not exhaustive):
- Pinmap
- Specs, levels, and Timing
- Patterns
- HRAM
- Source Memory
- Capture memory
- Seq. Flags and Registers
- PPMU
- Read/Write static
- Freq. Counter
- Clock Gen.
- Triggers
- Multi-instrument
- Utility

**Comment by sbethur** [comment]
All the required system tests have been created and executed before [0.9.0](https://github.com/ni/nimi-python/releases/tag/1.3.0) release.
